### PR TITLE
Use ::class where possible

### PR DESCRIPTION
**Pull request by malarzm**
This isn't even my final form

### CODE REVIEW

File: tests/Doctrine/ODM/MongoDB/Tests/Functional/Ticket/GH936Test.php [hunk 1]
```diff
@@ -24,25 +24,23 @@ public function testRemoveCascadesThroughProxyDocuments()
         $this->dm->flush();
         $this->dm->clear();
 
-        $foo = $this->dm->find(GH936Document::CLASSNAME, $foo->id);
+        $foo = $this->dm->find(GH936Document::class, $foo->id);
 
-        $this->assertInstanceOf('Doctrine\ODM\MongoDB\Proxy\Proxy', $foo->ref);
+        $this->assertInstanceOf(\Doctrine\ODM\MongoDB\Proxy\Proxy::class, $foo->ref);
```
Comment (carusogabriel) [inline tests/Doctrine/ODM/MongoDB/Tests/Functional/Ticket/GH936Test.php]: CS is breaking here. This should be referenced via `use` :+1:

Comment (malarzm) [reply tests/Doctrine/ODM/MongoDB/Tests/Functional/Ticket/GH936Test.php]: Thanks for catching :+1:

**Comment by malarzm** [comment]
> What did you use to do it? Something we can apply to Doctrine's CS?

@carusogabriel sadly it was manual work

**Comment by malarzm** [comment]
Chances are this is everything: some occurrences were left on purpose, some might have been missed. @carusogabriel I did separate commit on purpose in case you were reading the changes, after accepting I'll squash commits

**Pull request merged**